src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -502,6 +502,19 @@ private function isInteractiveInput($inputStream): bool
      */
     private function readInput($inputStream, Question $question): string|false
     {
+        if (null !== $question->getTimeoutSeconds() && $this->isInteractiveInput($inputStream)) {
+            $read = [$inputStream];
+            $write = null;
+            $except = null;
+            $timeoutSeconds = $question->getTimeoutSeconds();
+            $changedStreams = stream_select($read, $write, $except, $timeoutSeconds);
+
+            if (0 === $changedStreams) {
+                $plural = 1 === $timeoutSeconds ? '' : 's';
+                throw new MissingInputException("Timed out after waiting for input for $timeoutSeconds second$plural.");
```

Comment:
```suggestion
                throw new MissingInputException(\sprintf('Timed out after waiting for input for %d %s.', $timeoutSeconds, $timeoutSeconds === 1 ? 'second' : 'seconds'));
```
?

Reply:
Yes, all exceptions in Symfony are using `sprintf()`.

Reply:
Yes, all exceptions in Symfony are using `sprintf()`.

Reply:
Fixed.